QuizAPI/DTOs/QuestionDTO.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace QuizAPI.DTOs
+{
+	public class QuestionDTO
+	{
+		private string? question;
+		private string? answer;
+		private string? difficulty;
```

Comment:
Are these the names or ids how the difficulty/status/category? If its the id We need to change it to int

Reply:
No it's the string value of it difficulty: 'easy' | 'medium' | 'hard' etc

Reply:
is this file like a viewModel like it contains information about all the different tables so instead of displaying IDs in the original Questions table this one shows the actually values of the status, category and difficulty?

Reply:
So this is a DTO (Data Transfer Object) we're using it to pass through data in the patch request since only some data is needed for the question and not the all of it.

Reply:
ohhh I undstand now thank you